### ISSUE TITLE
Feature request: validation

### ISSUE THREAD

**Issue by rauwebieten** [panlatent/yii2-dotenv#7]
It would be nice to make use of the existing validation system of phpdotenv, for example:

    $dotenv->required('DATABASE_DSN');

Maybe make this available via the yii event system?

**Comment by panlatent** [comment]
Good idea. Where do we configure and call? I think the event system is a bad idea.
